Python/bltinmodule.c

```diff
@@ -1635,13 +1635,16 @@ anext as builtin_anext
     default: object = NULL
     /
 
-Return the next item from the async iterator.
+anext(aiterator[, default])
```

Comment:
```suggestion
async anext(aiterator[, default])
```

That way there's indication that results of `anext()` calls needs to be `await`ed.

I notice that, for example, we get
```
>>> help(asyncio.sleep)
Help on function sleep in module asyncio.tasks:

async sleep(delay, result=None)
    Coroutine that completes after a given time (in seconds).
```

Reply:
Good catch!  I think this is a good change, even though I can't find any other examples of this syntax showing up in a docstring.  +1 from me.

Reply:
Committed and reran the argument clinic.  Thanks @sweeneyde (and Go Bucks!)